lib/diagnostic.js

```diff
@@ -47,7 +47,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.fire.bind(alderaan);
```

Comment:
Partway there. `bind` only assigns an execution context to a function - it doesn't invoke that function.
To assign a context **AND** execute, you can use either `call` or `apply`. Or, alternatively, you could just invoke the newly bound function.

Also, the tests are expecting `fireOnAlderaan` to return a value - specifically, the return value from `fire`.